Cargo.toml

```diff
@@ -51,10 +51,10 @@ path-slash = "0.2"
 rayon = "1.8"
 clap = { version = "4", default-features = false, features = ["derive"] }
 rand = "0.8"
-polkavm-common = "0.10"
-polkavm-linker = "0.10"
-polkavm-disassembler = "0.10"
-polkavm = "0.10"
+polkavm-common = "0.12"
```

Comment:
I don't get it - this is already in `paritytech:main`? :D

Reply:
maybe this goes away after a rebase against main?

Reply:
This is part of a commit on the main branch that was accidentally included.

Reply:
Ok, can you please rebase so we get a clean diff?